src/unix/kqueue.c

```diff
@@ -34,6 +34,10 @@
 #include <fcntl.h>
 #include <time.h>
 
+#ifndef EV_OOBAND
+#define EV_OOBAND  EV_FLAG1
```

Comment:
How old?  We have a list of supported systems in SUPPORTED_PLATFORMS.md and I wouldn't want to add defines for platforms we don't currently support.